ansible/deploy_common.yml

```diff
@@ -32,3 +32,26 @@
   hosts: all
   roles:
     - {role: monit, tags: monit}
+
+- name: Configure static routes
+  hosts: all
+  tasks:
+
+  - name: Checks existing static routes
+    shell: 'ip route show {{ item.prefix }} | wc -l'
+    with_items: "{{ static_routes }}"
```

Comment:
This doesn't work if static_routes is undefined.

Reply:
Should it have a default value? what default value should it be?

Reply:
perhaps when: static_routes|default(false) ? i'll test it.